app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -24,7 +24,7 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => (
         return "edit/overview";
       }
 
-      if (!analysisAvailable(analysis)) {
+      if (analysisUnavailable(analysis)) {
```

Comment:
To be clear, this is what fixes the redirection bug, we have to check for `analysis` truthiness.